internal/docstore/firedocstore/fs.go

```diff
@@ -12,26 +12,150 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package firedocstore provides a docstore implementation backed by GCP
+// Firestore.
+// Use OpenCollection to construct a *docstore.Collection.
+//
+// URLs
+//
+// For docstore.OpenCollection, firedocstore registers for the scheme
+// "firedocstore".
```

Comment:
Same as other PRs: "firestore" is probably fine.

Reply:
Done.